second.txt

```diff
@@ -0,0 +1,2 @@
+- What is your name?
```

Comment:
remove these files from PR
https://www.youtube.com/watch?v=b2Qlqva4-H4&list=PL7FuXFaDeEX2JdShqx3URwdvqf6ow8G7H&index=14&t=351s&ab_channel=Mateacademy

Reply:
watch this video please